fw/sock.c

```diff
@@ -1588,6 +1590,57 @@ ss_synchronize(void)
 	return true;
 }
 
+/* This includes L2-L4 headers len + TLS record len (5 bytes) */
+#define TRANS_HDRS_SIZE			100
+/* http/2 HEADERS frame size */
+#define H2_HEADERS_FRAME_SIZE		13
```

Comment:
What I meant here is the whole frame size (frame header + 4 mandatory pseudo-headers as per RFC 9113 8.3.1)

Reply:
As for the warnings that you got, i believe it's something else since the return code is `-9`. Still need to investigate what is it, but in a separate task i guess.